sorc/install_rtma3d_all.sh

```diff
@@ -28,7 +28,9 @@ fi
 
 if [ $INSTALL_rtma3d_wrfarw = yes ] ; then
 
-cp -p ${BASE}/rtma3d_wrfarw.fd/WRFV3.9/main/wrf.exe ../exec/rtma3d_wrfarw_fcst
+# cp -p ${BASE}/rtma3d_wrfarw.fd/WRFV3.9/main/wrf.exe ../exec/rtma3d_wrfarw_fcst
```

Comment:
Can these be renamed for simplicity? E.g., keep the original file name "rtma3d_wrfarw_fcst" and the executable with no model forecast would then be named as "rtma3d_wrfarw_no_fcst".

Reply:
@MatthewMorris-NOAA 
I followed your suggestion and modified the filename of the wrf exe in scripts.
Thank you!